gli/markdown_file_cora.txt

```diff
@@ -0,0 +1,45 @@
+# **Graph Metrics**
```

Comment:
The format looks good. But we do not need to merge an example file into the repo. Please remove this file.